src/components/common/Actions.vue

```diff
@@ -4,7 +4,7 @@
     <div class="buttons-left">
       <v-btn
         v-if="isSummaryStep"
-        id="app-summary-cancel-btn"
+        id="app-cancel-btn"
```

Comment:
Yes, there's another button with this same ID on this page, but they're never displayed at the same time, so I think it's easier to have a single ID. Thoughts?